templates/panda/PolicyPanda.java

```diff
@@ -365,7 +371,7 @@ private ArrayNode findRepositoriesWithCLA(List<GHRepository> repos) {
         for (GHRepository repo : repos) {
             try {
                 // Look for CLA files with various naming patterns
-                GHContent cla = findFileInRepo(repo, ".*(?:cla|contributor|agreement).*");
+                GHContent cla = findFileInRepo(repo, ".*(?:cla|contributor(?!s)|agreement).*");
```

Comment:
Avoids catching files such as `CONTRIBUTORS.md` which by convention list those people who have contributed to a project.

Reply:
--asset-discovery and --org-settings are optional / additional.

There is no requirement that listed owners are publicly associated with the project (technically). GitHub organization membership is private by default, unless you go find that toggle.